### ISSUE TITLE
Updating Copyright Dates in Files

### ISSUE THREAD

**Issue by amarkpayne** [ReactionMechanismGenerator/RMG-Py#1031]
I have noticed that some of the RMG files have old dates listed in the copyright notice above the doc-string at the top of the file. For example, draw.py in rmgpy/molecule/ has the dates listed as 2009-2011. Is this correct or does the 2011 need to be changed to 2017? Do we have any sort of policy about this (i.e. "if you edit the file, update the date")? If not I don't mind doing the grunt work to update these files whenever I get free moments throughout the day.

**Comment by mliu49** [comment]
The headers are also slightly inconsistent across different files: there are differences in copyright years and authors and some files don't have a header at all.

I think we should probably update them and make it consistent across all files and the main `COPYING.txt` file. I think we could also consider renaming that file to `LICENSE.txt` so that GitHub recognizes it and puts it in the info bar.

**Comment by amarkpayne** [comment]
I think that this would probably be easy to implement using a python script to update the headers on the files. If not it wouldn't be so terrible to do by hand. I'll make a new branch and see what I can do in my free time.

**Comment by amarkpayne** [comment]
I think I have updated most of the RMG files on my UpdateCopyright branch. Specifically I targeted files in the rmgpy directory and subdirectories, though some data and input files I was unsure about. The major thing I saw was that some scripts had other author information on them. For example, rmgpy/cantherm/__init__.py is copyrighted by Josh Allen. Should this be changed? Files in similar situations are listed below if you are curious:

rmgpy/molecule/adjlist.py
rmgpy/molecule/adjlistTest.py
rmgpy/molecule/atomtype_database.py
rmgpy/molecule/atomtype_databaseTest.py

rmgpy/qm/qmverifier.py
rmgpy/qm/symmetry.py

rmgpy/transport.py
